Source/ASTextNode.mm

```diff
@@ -264,6 +264,28 @@ - (void)didLoad
   }
 }
 
+- (BOOL)supportsLayerBacking
+{
+  if (!super.supportsLayerBacking) {
+    return NO;
+  }
+  
+  // If the text contains any links, return NO.
+  NSAttributedString *attributedText = self.attributedText;
```

Comment:
Is their any performance concerns that this is called when setting layer backed to YES? Is it called in other paths?